frontend/i18next.scanner/en/translation.json

```diff
@@ -176,5 +176,9 @@
   "Invalid email": "Invalid email",
   "Invalid password": "Invalid password",
   "Error while saving your language": "Error while saving your language",
+  "Use this link to integrate this shared space's agenda to your": "Use this link to integrate this shared space's agenda to your",
+  "Use this link to integrate Tracim in your file explorer": "Use this link to integrate Tracim in your file explorer",
+  "CalDAV compatible software": "CalDAV compatible software",
+  "(protocole WebDAV)": "(WebDAV protocol)"
```

Comment:
There is a little typo that we missed during merge conflict, this line doesn't has the trailing comma ',' which makes the json invalid